ext/iconv/tests/iconv_basic_001.phpt

```diff
@@ -4,12 +4,6 @@ Test the basics to function iconv.
 Rodrigo Prado de Jesus <royopa [at] gmail [dot] com>
 --EXTENSIONS--
 iconv
```

Comment:
@iluuu1994 this test is failing even in regular linux CI, should some lib/locale be installed?